test/WebSites/FormatterWebSite/Controllers/StreamController.cs

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.IO;
+using Microsoft.AspNet.Mvc;
+using Microsoft.Net.Http.Headers;
+
+namespace FormatterWebSite
+{
+    public class StreamController : Controller
+    {
+        [HttpGet]
+        public Stream SimpleMemoryStream()
+        {
+            return CreateDefaultStream();
+        }
+
+        [HttpGet]
+        public Stream MemoryStreamWithContentType()
+        {
+            Response.ContentType = "text/html";
```

Comment:
Kind of useless to add an extra test. This is already automatically converted and the test for `ObjectResult` scenario is covered with the `[Produces]`.

We can have a separate discussion about a need for a `StreamResult`.